src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -89,33 +89,27 @@
+
+            "<p>This tool attempts to estimate library complexity from sequence of read pairs alone. Does so by sorting all reads by the first N bases (5 by default) of each read and then comparing reads with the first N bases identical to each other for duplicates.  Reads are considered to be duplicates if they match each other with no gaps and an overall mismatch rate less than or equal to MAX_DIFF_RATE (0.03 by default). " +
+            " Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering removes reads with any no-calls 'N' in the first N bases or with a mean base quality lower than MIN_MEAN_QUALITY across either the first or second read.  Unpaired reads are ignored in this computation. " +
             "" +
-            "<br /><br />PCR-induced duplicates can result from: inadequate amounts of starting material (genomic DNA, cDNA, etc.), " +
-            "losses during cleanups and size selection, and amplification-associated artifacts.   These duplicates " +
-            "are removed via the creation of a histogram consisting of the numbers of reads in duplicate set vs. the numbers " +
-            "of duplicate sets.  All bins that contain exactly one duplicate set are then removed from the histogram prior" +
-            " to estimating library size.<br /> <br />" +
-            "" +
-            "Optical duplicates result from reads that have identical sequences and whose clusters are proximal to each other.  " +
-            "These are identified using both the READ_NAME_REGEX string and the values for the OPTICAL_DUPLICATE_PIXEL_DISTANCE.<br /> <br />" +
-            "" +
-            "Sequence identity is determined by scanning the first five (default value) bases of a read.  Sequences are considered identical if" +
-            " they match each other with no gaps, have an overall mismatch rate less than or equal to the MAX_DIFF_RATE" +
-            " (0.03 is the default value), and the cluster proximity is less than the defined value (default is set at 100 pixels)." +
-            "" +
-            "<br /><br />Finally, the algorithm filters out poor quality reads defined as those with ambiguous base calls \"N\"s in the first 5 (default value) bases, " +
-            "reads with a mean base quality score lower than MIN_MEAN_QUALITY (20 is the default value), or any unpaired reads.   " +
+            "<p>The algorithm attempts to detect optical duplicates separately from PCR duplicates and excludes these in the calculation of library size. " +
+            "Also, since there is no alignment to screen out technical reads, one further filter is applied on the data.  After examining all reads a histogram is built of [#reads in duplicate set -> #of duplicate sets].   All bins that contain exactly one duplicate set are then removed from the histogram as outliers before library size is estimated.</p> " +
```

Comment:
"one further filter is applied on the data"  -> add "as follows." Otherwise it's not clear until two sentences later that this actually gets explained.